x-pack/plugins/apm/public/components/app/profiling_overview/index.tsx

```diff
@@ -17,7 +17,8 @@ import {
 } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { useKibana } from '@kbn/kibana-react-plugin/public';
-import React, { useMemo } from 'react';
+import React, { useMemo, useState } from 'react';
+import { SearchBarFilter } from '@kbn/observability-shared-plugin/public';
```

Comment:
Is the `SearchBarFilter` component designed with a generic purpose, or is it specifically for profiling? if it's the latter wdyt of renaming to `ProfilingSearchBarFilter`

Reply:
for now, I don't see any other use than Profiling. Renaming it as suggested.